app/src/main/java/org/mozilla/fenix/onboarding/view/OnboardingPage.kt

```diff
@@ -0,0 +1,127 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.onboarding.view
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.button.PrimaryButton
+import org.mozilla.fenix.compose.button.SecondaryButton
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * A page for displaying Onboarding Content.
+ *
+ * @param onboardingPageUiState The page content that's displayed.
+ * @param onPrimaryButtonClick Invoked when the user clicks the primary button.
+ * @param onSecondaryButtonClick Invoked when the user clicks the secondary button.
+ * @param modifier The modifier to be applied to the Composable.
+ */
+@Composable
+fun OnboardingPage(
+    onboardingPageUiState: OnboardingPageUiState,
+    onPrimaryButtonClick: () -> Unit,
+    onSecondaryButtonClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier.semantics(mergeDescendants = true) {},
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.SpaceEvenly,
+    ) {
+        // Since the vertical arrangement is SpaceEvenly, the text and buttons are wrapped
+        // their own containing Composables so the OuterColumn only has 3 children with
+        // space in between and around them
+        // Item 1
```

Comment:
I think we could remove comments like these. I would hope it's generally understood when someone sees `verticalArrangement = Arrangement.SpaceEvenly` rather than have to explain what a property of `verticalArrangement` is doing.